src/lib/viewers/controls/rotate/__tests__/RotateControl-test.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import RotateControl from '../RotateControl';
+import IconRotate24 from '../../icons/IconRotate24';
```

Comment:
By convention our import order is: absolute imports > relative imports > stylesheets -- with each category alphabetized

Reply:
Got it, thanks!